### ISSUE TITLE
Do moderators need an option to at will ban a user from making reports for time or until they appeal?

### ISSUE THREAD

**Issue by CJH87** [bethylamine/soupcan#137]
In composing #136 I dreamed up a horrible scenario.

What countermeasures exist for moderators to respond to a malicious actor controlling three or more accounts totalling 100% trust?

If it's two that's easy as the appeal function would shut them down immediately as either of the accounts is appealed.

If a malicious actor acquired three acounts with a total trust score over 100%, they could potentially go undetected, flooding the database with irelevant non-transphobic red users.

In the event tha they were noticed, could a single moderator reduce the trust scores of three accounts faster than those three accounts can regain trust? 

This post may give away details of how the trust system works and may need to be censored by @bethylamine .

In short, perhaps moderators need an option to block an account from making reports if they are sufficiently suspicious of the account's acitivty. 

I've been unable to access the queue for a while and I've just noticed some users have more than 50%. So I'm a little OOTL on that.
